community/index.md

```diff
@@ -80,21 +87,76 @@ Various Julia videos and playlists are posted on [YouTube](https://www.youtube.c
+* [Julia StackOverflow](https://twitter.com/JuliaOverflow)
+* [JuliaComputing](https://twitter.com/JuliaComputing)
+* [Julia-i18n](https://twitter.com/julia_i18n)
+* [JuliaLangEs](https://twitter.com/JuliaLangsEs) use [#JuliaLangEs](https://twitter.com/hashtag/JuliaLangEs?src=hash) hashtag.
+* [JuliaLang Japan](https://twitter.com/JuliaLangJa) use [#JuliaTokyo](https://twitter.com/hashtag/JuliaTokyo?src=hash) hashtag.
 
 # StackOverflow
 
-Use the [julia-lang](http://stackoverflow.com/questions/tagged/julia-lang) tag on StackOverflow, if you post a Julia related question on StackOverflow.
+* Use the [julia-lang](http://stackoverflow.com/questions/tagged/julia-lang) tag on StackOverflow, if you post a Julia related question on StackOverflow.
+* Use the [julia](http://es.stackoverflow.com/questions/tagged/julia) tag on *StackOverflow en Español*, if you post a Julia related question on StackOverflow in Spanish.
 
 # Chat Rooms
 
+## IRC
+
 * Many Julia users and developers hang out in the [#julia IRC channel](http://webchat.freenode.net/?channels=julia) on Freenode.
+
+## Gitter
+
 * [julia](https://gitter.im/JuliaLang/julia) Gitter chat room is for discussions around the usage of Julia.
-* [julialang-es](https://gitter.im/JuliaLangEs/julialang-es) Gitter chat room is for discussions around the usage of Julia in the Spanish language.
 * [julia-coverage](https://gitter.im/kshyatt/julia-coverage) Gitter chat room is for discussions around Julia tests coverage.
-
-# QQ groups
+* [JuliaPraxis](https://gitter.im/JuliaPraxis) gitter chat rooms are for discussions around Julia best practices.
+* [JuliaQuantum](https://gitter.im/JuliaQuantum/home)
+* [JunoIDE](https://gitter.im/JunoLab/Juno)
+* [JuliaML](https://gitter.im/JuliaML/chat)
+* [JuliaGL](https://gitter.im/JuliaGL/meta)
+* [BioJulia](https://gitter.im/BioJulia/home)
+* [JuliaOpt](https://gitter.im/JuliaOpt/home)
+* [IntelLabs](https://gitter.im/IntelLabs/home)
+* [JuliaOpt](https://gitter.im/JuliaOpt/home)
+* [UCIDataScienceInitiative](https://gitter.im/UCIDataScienceInitiative/home)
+* [Julia-i18n](https://gitter.im/Julia-i18n/julia-i18n) Gitter chat room is for discussions around Julia internationalization and localization.
+* [julialang-es](https://gitter.im/JuliaLangEs/julialang-es) Gitter chat room is for discussions around the usage of Julia in the Spanish language.
+* [JuliaLangPt/julia](https://gitter.im/JuliaLangPt/julia) Gitter chat room is for discussions around the usage of Julia in the Portuguese language.
+
+### Packages
+
+* [Gadfly.jl](https://gitter.im/dcjones/Gadfly.jl)
+* [LightGraphs.jl](https://gitter.im/JuliaGraphs/LightGraphs.jl)
+* [GR.jl](https://gitter.im/jheinen/GR.jl)
+* [FixedSizeArrays.jl](https://gitter.im/SimonDanisch/FixedSizeArrays.jl)
+* [DSGE.jl](https://gitter.im/FRBNY-DSGE/DSGE.jl)
+* [Docile.jl](https://gitter.im/MichaelHatherly/Docile.jl)
+* [Mathematica.jl](https://gitter.im/MikeInnes/Mathematica.jl)
+* [Nanopore.jl](https://gitter.im/gawbul/Nanopore.jl)
```

Comment:
Nanopore.jl is an empty package and no updates for two years. So, I guess this is abandoned.